server/src/test/java/org/opensearch/common/lucene/LuceneTests.java

```diff
@@ -92,62 +92,12 @@
 import java.util.HashSet;
 import java.util.List;
 import java.util.Set;
-import java.util.concurrent.CountDownLatch;
-import java.util.concurrent.atomic.AtomicBoolean;
 
 import static org.hamcrest.Matchers.equalTo;
 
 public class LuceneTests extends OpenSearchTestCase {
     private static final NamedWriteableRegistry EMPTY_REGISTRY = new NamedWriteableRegistry(Collections.emptyList());
 
-    public void testWaitForIndex() throws Exception {
```

Comment:
Why this test is removed?

Reply:
The test is for `Lucene.waitForIndex()` which is not used anywhere.

Reply:
Fair enough, thank you